examples/getting-started/example1/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+project(powsybl-iidm4cpp-getting-started-1)
+
+find_package(Boost 1.56 REQUIRED COMPONENTS date_time filesystem program_options system unit_test_framework)
+find_package(LibXml2 REQUIRED)
+find_package(LibIIDM REQUIRED)
+
+add_executable(iidm4cpp_reader main.cpp)
+target_link_libraries(iidm4cpp_reader IIDM::iidm)
```

Comment:
Doing that way, you do not consider that the example is part of the project? Do you miss to remove it?